.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master, release]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 21 * * 5'
```

Comment:
Why do we need a cron for code review? What it starts and why on Friday?

Reply:
It will start github code scanner on schedule to identify the latest vulnerabilities and errors that GitHub, security researchers, and the community discover, even when we don't actively maintain the repository.